_scalingo_json_schema/index.md

```diff
@@ -180,7 +180,7 @@ and the value should have the following structure:
 | field    | type    | description                                                            |
 | -------- | ------- | ---------------------------------------------------------------------- |
 | amount   | integer | Number of containers to start once the application is deployed         |
-| size     | string  | Container size `name` ([get the list of sizes](/container-sizes.html)) |
+| size     | string  | Container size `name` ([get the list of sizes](https://doc.scalingo.com/platform/internals/container-sizes)) |
```

Comment:
We actually want to target the documentation page explaining the API endpoint to list container sizes.

```suggestion
| size     | string  | Container size `name` ([get the list of sizes](/container_sizes#list-the-container-sizes-available)) |
```